Makefile

```diff
@@ -98,9 +98,9 @@ srpm: source
 
 copr_build: srpm
 	@echo "--- Build RPM ${PKGNAME}-${VERSION}-${RELEASE}.el7.rpm in COPR ---"
-	@echo copr --config $(_COPR_CONFIG) build $(_COPR_REPO) \
+	@echo copr-cli --config $(_COPR_CONFIG) build $(_COPR_REPO) \
 		packaging/SRPMS/${PKGNAME}-${VERSION}-${RELEASE}*.src.rpm
-	@copr --config $(_COPR_CONFIG) build $(_COPR_REPO) \
+	@copr-cli --config $(_COPR_CONFIG) build $(_COPR_REPO) \
```

Comment:
why that change? I know the copr-cli is probably the name, but copr is available as well. At least in my case on my system. Just want to understand the need for the change @vojtechsokol  @jmikovic ?

Reply:
Explained. On the testing farm the copr-cli is installed via `pip` - so it's not installed rpm as we would expect in this case (e.g. `yum install copr-cli`) -> so the symlink does not exist as the python package doesn't care about that.

Reply:
Btw, regarding the gazillion problems around `pip`, in case it start to fail on the copr-cli installation, I suggest to be prepared for installation via `yum` or `dnf`.

Reply:
Yep, the issue was that `copr` command is not available when installing from pip. When installed from rpm package, there are both (and `copr` being a symlink to `copr-cli`), so it should be safer to use `copr-cli`.

Reply:
And regarding using yum/dnf - the runner is ubuntu, so no system package for copr...